test/helpers.js

```diff
@@ -0,0 +1,97 @@
+import katex from "../katex";
+import ParseError from "../src/ParseError";
+import parseTree from "../src/parseTree";
+import Settings from "../src/Settings";
+
+export const defaultSettings = new Settings({
+    strict: false, // deal with warnings only when desired
+});
+export const strictSettings = new Settings({strict: true});
+
+export const _getBuilt = function(expr, settings) {
+    const usedSettings = settings ? settings : defaultSettings;
```

Comment:
I think we can use `||` and overwrite parameter, i.e., `settings = settings || defaultSetting`.